ctwrap/strategy.py

```diff
@@ -148,6 +153,52 @@ def load(cls, strategy, name=None):
 
         return type(cls_hook.__name__, (cls_hook, ), {})(value, name=name)
 
+    @staticmethod
+    def _parse_mode(strat_dict):
+        """
+        parse the strategy/variation values based on the mode specified
+
+        Argument:
+            strat_dict(dict): variation values specification
+
+        Output:
+            out (dict): parsed variation values
+
+        """
```

Comment:
Since this is a `@staticmethod` that does not depend on anything within the class, it may make sense to define it outside the class (see other *hidden* methods in this module, e.g. `_replace_entry`). Also, please add a unit-test (which will be easier for stand-alone methods also: see `test_strategy.py::TestReplace`).

Please take care of this first - I'll add other comments at a later point.